website/docs/r/cloud_service_epg.html.markdown

```diff
@@ -0,0 +1,92 @@
+
+
+## Example Usage ##
+
+```hcl
+resource "aci_cloud_service_epg" "example" {
+  cloud_application_container_dn  = aci_cloud_applicationcontainer.example.id
+  name                            = "example"
+  access_type                     = "Public"
+  deployment_type                 = "CloudNative"
+  flood_on_encap                  = "disabled"
+  label_match_criteria            = "AtleastOne"
+  cloud_service_epg_type          = "Azure-SqlServer"
+  cloud_rs_cloud_epg_ctx          = aci_vrf.example.id
+}
+```
+
+## Argument Reference ##
+
+* `cloud_application_container_dn` - (Required) Distinguished name of the parent Cloud Application container object. Type: String.
+* `name` - (Required) Name of the Cloud Service EPG object. Type: String.
+* `annotation` - (Optional) Annotation of the Cloud Service EPG object. Type: String.
+* `name_alias` - (Optional) Name Alias of the Cloud Service EPG object. Type: String.
+* `access_type` - (Optional) This refers to the type of connectivity to the service. It could be a public or private connectivity. Allowed values are "Private", "Public", "PublicAndPrivate", "Unknown", and default value is "Public". Type: String.
+* `azure_private_endpoint` - (Optional) Naming for Azure Private Endpoint created from the Service Cloud EPG. Naming override for any Azure Private Endpoint that gets created from this service EPG. The set of variable supported by the naming override is the same of those supported in the global naming policy. However, there is no mandatory variable enforced by validations. Type: String.
+* `custom_service_type` - (Optional) Custom Service type. A custom service type used when this EPG is use as custom service EPG with public or private access. As an e.g. this is used to provide the service tag for an Azure service. Type: string
+* `deployment_type` - (Optional) cloud service deployment type.deploymentType refers to the type of deployment of the service. It could be a PaaS service, a PaaS service deployed in the managed VNET/VPC, a SaaS service consumed or a SaaS service offered. Allowed values are "CloudNative", "CloudNativeManaged", "Third-party","Third-partyManaged", "Unknown", and default value is "Unknown". Type: String.
+* `flood_on_encap` - (Optional) Handling of L2 Multicast/Broadcast and Link-Layer traffic at EPG level. Control at EPG level if the traffic L2 Multicast/Broadcast and Link Local Layer should be flooded only on ENCAP or based on bridg-domain settings. Allowed values are "disabled", "enabled", and default value is "disabled". Type: String.
+* `label_match_criteria` - (Optional) Provider Label Match Criteria. Allowed values are "All", "AtleastOne", "AtmostOne", "None", and default value is "AtleastOne". Type: String.
+* `preferred_group_member` - (Optional) Represents parameter used to determine if the Cloud Service EPG is part of a group that does not a contract for communication. Allowed values are "exclude", "include", and default value is "exclude". Type: String.
```

Comment:
->  Represents the parameter used to determine if the Cloud Service EPG is part of a group that does not have a contract for communication. Allowed values are "exclude", "include", and default value is "exclude". Type: String.